commands/db.bee.inc

```diff
@@ -119,8 +119,14 @@ function db_export_bee_callback($arguments, $options) {
     $extra = '--no-tablespaces ';
   }
 
+  $export_command = executable_for('mysqldump');
+  if (is_null($export_command)) {
+    bee_message("The MySQL export command 'mysqldump' cannot be found in " .
+                "this system. Please install it and try again.", 'error');
```

Comment:
Within Backdrop, there is a function called `t()` that encloses translatable strings.  In bee, because we can't always rely on Backdrop being bootstrapped, we have our own function called `bt()` for translatable strings. Translatable strings are an exception (there are a couple of others) to the 80 character rule and the translatable string should be in a single string.

```suggestion
    bee_message(bt("The MySQL export command 'mysqldump' cannot be found in this system. Please install it and try again."), 'error');
```